app/services/item_service.rb

```diff
@@ -0,0 +1,25 @@
+class ItemService
+  def self.all_items
+    content = conn.get("/api/v1/items")
+
+    results = (parse_response(content.body))[:data]
+
+    results
+  end
+
+  def self.item_details(id)
+    content = conn.get("/api/v1/items/#{id}")
+
+    results = (parse_response(content.body))[:data]
+
+    results
+  end
+
+  def self.parse_response(response)
+    JSON.parse(response, symbolize_names: true)
+  end
+
+  def self.conn
+    Faraday.new(url: "http://localhost:3000")
+  end
+end
```

Comment:
👍

Reply:
This goes for the MErchantService too.. but don't go into the :data root key within the service. Let your service be responsible for calling and getting back ALL data -- If you had another facade that wanted data outside of the data root key, then you just limited that in the service. Let your facades manage the root keys that you need to filter through.